java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -668,6 +668,8 @@ private static Object[] serverManagementCommands(BaseTransaction<?> transaction)
                 .lolwut(1)
                 .flushall()
                 .flushall(ASYNC)
+                .flushdb()
+                .flushdb(ASYNC)
```

Comment:
This test will fail if we run against 6.1 or prior

Reply:
> Starting with Redis version 6.2.0: Added the SYNC flushing mode modifier.